diff --git a/CMakeLists.txt b/CMakeLists.txt
index 294655fa1..28765adb3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,6 +6,9 @@
   cmake_policy(SET CMP0167 OLD)
 endif()
 
+# use gnu standard install directories
+include(GNUInstallDirs)
+
 find_package(Kokkos 4.2 REQUIRED CONFIG)
 message(STATUS "Found Kokkos: ${Kokkos_DIR} (version \"${Kokkos_VERSION}\")")
 
@@ -74,12 +77,12 @@
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src/interpolation>
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src/interpolation/details>
   $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
-  $<INSTALL_INTERFACE:include/ArborX>
-  $<INSTALL_INTERFACE:include/ArborX/details>
-  $<INSTALL_INTERFACE:include/ArborX/geometry>
-  $<INSTALL_INTERFACE:include/ArborX/kokkos_ext>
-  $<INSTALL_INTERFACE:include/ArborX/interpolation>
-  $<INSTALL_INTERFACE:include/ArborX/interpolation/details>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/ArborX>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/ArborX/details>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/ArborX/geometry>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/ArborX/kokkos_ext>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/ArborX/interpolation>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/ArborX/interpolation/details>
 )
 
 install(TARGETS ArborX
@@ -89,7 +92,7 @@
 
 install(EXPORT ArborXTargets
   NAMESPACE ArborX::
-  DESTINATION lib/cmake/ArborX
+  DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake/ArborX
 )
 
 set(ARBORX_VERSION_STRING "1.7")
@@ -124,7 +127,7 @@
 include(CMakePackageConfigHelpers)
 configure_package_config_file(cmake/ArborXConfig.cmake.in
   ${CMAKE_CURRENT_BINARY_DIR}/ArborXConfig.cmake
-  INSTALL_DESTINATION lib/cmake/ArborX
+  INSTALL_DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake/ArborX
 )
 write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/ArborXConfigVersion.cmake
   VERSION ${ARBORX_VERSION_STRING}
@@ -134,17 +137,17 @@
   ${CMAKE_CURRENT_BINARY_DIR}/ArborXConfig.cmake
   ${CMAKE_CURRENT_BINARY_DIR}/ArborXConfigVersion.cmake
   ${CMAKE_CURRENT_BINARY_DIR}/ArborXSettings.cmake
-  DESTINATION lib/cmake/ArborX )
+  DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake/ArborX )
 
 if(ARBORX_ENABLE_MPI)
-  install(DIRECTORY ${PROJECT_SOURCE_DIR}/src/ DESTINATION include/ArborX
+  install(DIRECTORY ${PROJECT_SOURCE_DIR}/src/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/ArborX
           FILES_MATCHING PATTERN "*.hpp")
 else()
-  install(DIRECTORY ${PROJECT_SOURCE_DIR}/src/ DESTINATION include/ArborX
+  install(DIRECTORY ${PROJECT_SOURCE_DIR}/src/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/ArborX
           FILES_MATCHING PATTERN "*.hpp"
           PATTERN "*Distribut*" EXCLUDE)
 endif()
-install(DIRECTORY ${PROJECT_BINARY_DIR}/include/ DESTINATION include/ArborX
+install(DIRECTORY ${PROJECT_BINARY_DIR}/include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/ArborX
         FILES_MATCHING PATTERN "*.hpp")
 
 if(NOT CMAKE_BUILD_TYPE)
